### PR TITLE
Add German translation

### PR DESCRIPTION
**Pull request by 0xhtml**

### CODE REVIEW

File: po/de.po [hunk 1]
```diff
@@ -19,60 +19,60 @@ msgstr ""
 
 #: src/MainWindow.vala:54
 msgid "Image Optimizer"
-msgstr ""
+msgstr "Bildoptimierung"
```
Comment (GijsGoudzwaard) [inline po/de.po]: This shouldn't be translated, since this is the name of the application.

File: po/extra/de.po [hunk 2]
```diff
@@ -21,87 +21,97 @@ msgstr ""
 #: data/com.github.gijsgoudzwaard.image-optimizer.desktop.in:3
 #: data/com.github.gijsgoudzwaard.image-optimizer.desktop.in:4
 msgid "Image Optimizer"
-msgstr ""
+msgstr "Bildoptimierung"
```
Comment (GijsGoudzwaard) [inline po/extra/de.po]: This shouldn't be translated, since this is the name of the application.

**Comment by GijsGoudzwaard** [comment]
Thanks for your contribution! I added a couple of comments, could you take a look at these?

**Comment by 0xhtml** [comment]
OK, I fixed the translated application name.

**Comment by GijsGoudzwaard** [comment]
Thanks. Looks good!

**Pull request merged**